### DISCUSSION TITLE
JNDIObjectLocatorBuilder syntax

### DISCUSSION THREAD

**Discussion by heatherita** [flowlogix/flowlogix#312] [Q&A] [answered]
Hi Lenny.

Does this from the old `com.flowlogix.ejb.JNDIObjectLocator `

`String appPrefix = System.getProperty(EMAILSRVR_APP_PREFIX, "java:global/my-apps/myapp");  
 locator = new JNDIObjectLocator(appPrefix);`

become this?

 `locator = JNDIObjectLocator.builder().namespace(appPrefix).build();`

or this?

  `locator = JNDIObjectLocator.builder().initialHost(appPrefix).build();`

**Comment by lprimak** [comment, marked as answer]
Quick look at the API: https://omnifaces.org/docs/javadoc/3.13/org/omnifaces/util/JNDIObjectLocator.JNDIObjectLocatorBuilder.html
yields this:
`builder().global().namespace()`
